NDB.Covid19/NDB.Covid19.Droid/OAuth2/AuthUrlSchemeInterceptorActivity.cs

```diff
@@ -50,6 +51,9 @@ protected override void OnCreate(Bundle savedInstanceState)
                 // load redirect_url Page for parsing
                 AuthenticationState.Authenticator.OnPageLoading(uri_netfx);
 
+                Intent intent = new Intent(this, typeof(QuestionnairePageActivity));
```

Comment:
have you checked what will happen if user will cancel login?

Reply:
Do you mean if browser is closed? If, so then the user is back at the consent page where they started the auth process